Gifski/util.swift

```diff
@@ -348,6 +349,17 @@ extension AVAssetImageGenerator {
 					)
 				)
 			case .failed:
+				// TODO: Ideally, we should trim blank frames when initially reading the video in `VideoValidator.swift`, but I don't know a way to detect blank frames.
```

Comment:
I think this workaround is actually a pretty good fix, maybe even better than trimming the video as this covers the case when a video has a blank frame in the middle of a video? not sure if possible though

Reply:
Yes, I would still keep this fix even if I find a way to trim. The reasons I want to trim is that then it wouldn't waste time trying to decode empty frames and also because the empty frames show up in the editor preview.

Reply:
> video has a blank frame in the middle of a video

It is possible if someone has composed multiple videos in a composition but has not done the timing correctly and left a gap between them.